src/components/TagSection/TagSection.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useContext, useEffect } from "react";
+
+/* Components */
+import { EuiComboBox } from "@elastic/eui";
+
+/* Styles */
+import "./_tag-section.scss";
+
+/* Models */
+import { Tag } from "../../models/Tag";
+
+/* Store */
+import FirebaseContext from "../../firebase/context";
+
+export interface TagSectionProps {
+  initial_tags: Array<Tag>;
+  className: string;
+}
+
+export function TagSection(props) {
+  const firebase = useContext(FirebaseContext);
+  const [selectedOptions, setSelected] = useState(props.initial_tags);
+  const [options, setOptions] = useState(Array<any>());
+
+  const fetchTags = async () => {
+    let x = await firebase?.getTags();
```

Comment:
instead of doing this do
```
let tags = x.map((doc) => {
     return doc.data().value
});
```

Reply:
Since it's not actually an array it doesn't have the map function available